patches/server/1065-Create-Limited-set-for-Entity.patch

```diff
@@ -0,0 +1,74 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Intybyte <vaan.testwork@gmail.com>
+Date: Mon, 30 Sep 2024 17:16:36 +0200
+Subject: [PATCH] Create Limited set for Entity
+
+
+diff --git a/src/main/java/io/papermc/paper/entity/PaperLimitedSet.java b/src/main/java/io/papermc/paper/entity/PaperLimitedSet.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..456c8cdb38d29a73a96dc44f7e8e4ce799cbab38
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/entity/PaperLimitedSet.java
```

Comment:
Lets put this in some `util` package or smth. Instead of `entity`. Also, just call it like `SizeLimitedSet`, Paper doesn't really apply to it.